pkgs/servers/misc/subsonic/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, jre }:
+
+let version = "5.2.1"; in
+
+stdenv.mkDerivation rec {
+  name = "subsonic-${version}";
+  inherit version;
+
+  src = fetchurl {
+    url = "mirror://sourceforge/subsonic/subsonic-${version}-standalone.tar.gz";
+    sha256 = "523fa8357c961c1ae742a15f0ceaabdd41fcba9137c29d244957922af90ee791";
+  };
+
+  inherit jre;
+
+  builder = ./builder.sh;
+
+  meta = {
+    homepage = http://subsonic.org;
+    description = "Subsonic";
```

Comment:
Bad description. Give a short sentence about this pkg. And omit pkg name in the description.